test/packages/profiling_symbolizer/data_stream/example/manifest.yml

```diff
@@ -1,8 +1,13 @@
-title: "Universal Profiling Symbols"
+title: "Example data stream of the profiling type"
 type: profiling
-release: beta
 streams:
-  - input: pf-elastic-symbolizer
-    title: Universal Profiling Symbols
-    description: "Collect debug symbols for native binaries"
-    enabled: true
+  - input: logfile
+    title: Sample logs
+    description: Collect sample logs
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        default:
+          - /var/log/*.log
```

Comment:
Not sure why we are removing this?

Reply:
I recreated the package from scratch to fill some missing files and forgot to recover this content. Fixed.